modules/contacts/site.js

```diff
@@ -260,3 +260,10 @@ var check_cc_exist_in_contacts_list = function() {
     }
     return "";
 };
+// Initialize Bootstrap tooltips for contact names
+$(document).ready(function() {
+    var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
+    var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+        return new bootstrap.Tooltip(tooltipTriggerEl);
```

Comment:
Shouldn't these be initialized by default when the page loads and bootstrap initializes? I don't think it is a good idea to manually initialize them here.

Reply:
You're right.

 Bootstrap tooltips didn’t auto-init. Because Cypht uses its own system, I replaced them with native tooltips.